pkg/process/util/network_filter.go

```diff
@@ -44,25 +49,68 @@ func ParseConnectionFilters(filters map[string][]string) (blacklist []*Connectio
 
 		validFilter := true
 		for _, v := range ports {
-			if v == "*" {
+			// reset validFilter for each port
+			validFilter = true
+			v = strings.ToUpper(v)
+			// set tcp and udp to true to prevent exclusion by default
+			tcp, udp := true, true
+			switch {
+			case strings.HasPrefix(v, "TCP"):
+				udp = false
+				v = strings.TrimPrefix(v, "TCP")
+			case strings.HasPrefix(v, "UDP"):
+				tcp = false
+				v = strings.TrimPrefix(v, "UDP")
+			}
+
+			if strings.ContainsRune(v, '*') {
```

Comment:
Can you do a  `v = strings.TrimSpace("")` and then a `v == "*"` instead of using `ContainsRune`? I think this will accept  `TCP * somethingelse`    as a valid filter